lld/test/MachO/objc-category-conflicts.s

```diff
@@ -1,3 +1,6 @@
+# `-no_objc_relative_method_lists` needs to be explicitly added to this test to avoid crashing after `-objc_relative_method_lists` was made default.
+# TODO: Make this test compatible with default `-objc_relative_method_lists` and remove the `-no_objc_relative_method_lists` flag. Issue #101419
```

Comment:
No need for this - we'll just leave everything `-no_objc_relative_method_lists` forever - the tests rely on this format output anyways.

Reply:
Opps, forgot to remove unnecessary comments.